src/core/utils/default_app_categories.ts

```diff
@@ -29,20 +29,28 @@ export const DEFAULT_APP_CATEGORIES = Object.freeze({
     euiIconType: 'logoKibana',
     order: 1000,
   },
+  enterpriseSearch: {
+    id: 'enterpriseSearch',
+    label: i18n.translate('core.ui.enterpriseSearchNavList.label', {
```

Comment:
Where are these values read from?

Reply:
https://github.com/elastic/kibana/pull/66922/files#diff-913b8ed1c6daf67597bca2d36482a66c

We simply define the JSON keys, another team is in charge of the translations from my understanding

Reply:
Ah, got it.